implementations/rust/ockam/ockam_command/src/project/addon/disable.rs

```diff
@@ -53,14 +57,22 @@ async fn run_impl(
     } = cmd;
 
     let mut rpc = Rpc::embedded(&ctx, &opts).await?;
-    let endpoint = format!(
-        "{}/{}",
-        base_endpoint(&opts.config.lookup(), &project_name)?,
-        addon_id
-    );
-    let req = Request::delete(endpoint).body(CloudRequestWrapper::bare(controller_route));
+    let body = DisableAddon::new(addon_id);
+    let endpoint = disable_addon_endpoint(&opts.config.lookup(), &project_name)?;
+
+    let req = Request::post(endpoint).body(CloudRequestWrapper::new(
+        body,
+        controller_route,
+        None::<CowStr>,
+    ));
     rpc.request(req).await?;
-    rpc.is_ok()?;
+    let res = rpc.parse_response::<CreateOperationResponse>()?;
+    let operation_id = res.operation_id;
+
+    check_for_completion(&ctx, &opts, &cloud_opts, rpc.node_name(), &operation_id).await?;
+
+    println!("Addon disabled successfully");
```

Comment:
For outputting command results to a user; we use `opts.terminal`

In this case you can use

```rs
opts.terminal.write_line(&fmt_ok!("Addon disabled successfully"))?;
```

Reply:
Done in https://github.com/build-trust/ockam/pull/4781/commits/55e6744a165eb8693b7a82c934d325b745c53a72